voicevox_engine/metas/MetasStore.py

```diff
@@ -0,0 +1,50 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+from voicevox_engine.metas.Metas import SpeakerSupportPermittedSynthesisMorphing
+
+
+class MetasStore:
+    def __init__(self, engine_speakers_path: Path) -> None:
+        self._engine_speakers_path = engine_speakers_path
+        self._loaded_metas = {
+            folder.name: self.fill_missing_speaker_properties(
+                json.loads((folder / "metas.json").read_text(encoding="utf-8"))
+            )
+            for folder in engine_speakers_path.iterdir()
+        }
+
+    @staticmethod
+    def fill_missing_speaker_properties(speaker: Dict):
+        speaker.setdefault(
+            "supported_features",
+            dict(),
+        )
+        speaker["supported_features"].setdefault(
+            "permitted_synthesis_morphing",
+            SpeakerSupportPermittedSynthesisMorphing(None),
+        )
+        return speaker
+
+    def speaker_engine_metas(self, speaker_uuid: str):
+        return self.loaded_metas[speaker_uuid]
+
+    def combine_metas(self, core_metas):
```

Comment:
何の情報を受け取って何の情報を返すのか型書いておくと更に良いかもと思いました！
たぶんimportされてないのでエラーになると思うのですが、こんな感じですよね。
```suggestion
    def combine_metas(self, core_metas: CoreSpeaker) -> Speaker:
```

Reply:
`Speaker`に変換する前のjsonから変換したdictなので型としては違うんですよね...
内部で早めにModelに変換して扱ったほうがいいでしょうか

Reply:
あ、なるほどです！このクラスはDictのまま変換しまくる設計だということに気づきませんでした。

やっぱりDictのまま扱うのは結構簡単に不具合になるのでなるべく型に守られた世界に変換するほうが良さそうに思います。
でも今から書き直すとなると時間かかると思うんですよね･･･。
日曜中に変更とかが難しそうであれば、いったんマージするのが良いのかなと思いました！

Reply:
遅くなってしまいましたが先ほど気づいたので修正しました🙇‍♂️